### ISSUE TITLE
Please document the DNS/server config for the rooms sub-domaine names

### ISSUE THREAD

**Issue by Lucas-C** [martindale/soundtrack.io#186]
Hi,

I'm trying to setup an instance of soundtrack.io for me and some friends,
and I don't really know how to configure my server so that e.g. "coding.soundtrack.chezsoi.org" can be queried.
Could you please document this configuration ?

Thanks

**Comment by martindale** [comment]
This should be documented here: https://github.com/martindale/soundtrack.io/blob/soundtrack.io/README.md#testing-rooms

**Comment by Lucas-C** [comment]
Oh, ok. I thought this paragraph was only for tests, as it concerns only local hostname lookups.
I tried this but I still remain stuck with:

    $ curl https://coding.soundtrack.chezsoi.org
    curl: (6) Could not resolve host: coding.soundtrack.chezsoi.org

**Comment by martindale** [comment]
It looks like you're deploying to a public webserver – you'll need to use a wildcard entry at the apex domain, I believe.  I have not tested using a subdomain as the top-level site, as we have only ever deployed locally or on soundtrack.io.

I'm really excited that you're deploying a copy, by the way.  I'd be very interested to discuss a peer-to-peer data-sharing arrangement, even ahead of Fabric and Maki.  It'd be useful to attempt to synchronize / reconcile differences between our `Tracks` and `Artists` collections.

**Comment by Lucas-C** [comment]
> you'll need to use a wildcard entry at the apex domain, I believe.

Ok, but to what path on the Node server should I route the requests ?? Can I route them directly to the root `/`, and the app will detect the room from the HTTP "Host" header ?

I only plan to use this with a close circle of friends, and we may prefer to keep our playlists separated. But I'd be happy to give you some feedback on this experiment :)
Also, what are Fabric & Maki ?

**Comment by martindale** [comment]
Yes, the `/` route handles everything automatically, based on the `Host` field.  Based on the changes you've made, everything should work as expected, but let me know if it doesn't! :)

I'll connect with you in the Maki Slack regarding the other questions.

**Comment by Lucas-C** [comment]
In case anyone else faces the same issue, here is the solution I found to the DNS wildcard problem, considering I'm using `bind9` on my machine : simply add the following line into `/etc/bind/db.chezsoi.org` :

    *.soundtrack      IN              CNAME           soundtrack.chezsoi.org.

And then: `sudo service bind9 restart`.

Your Maki + Fabric project sounds great ! I'd be happy to help, I'll trying to get a look at those during the week
